js/slider.js

```diff
@@ -0,0 +1,21 @@
+
+var slideIndex = 0;
+showSlides();
+
+function showSlides() {
+  var i;
+  var slides = document.getElementsByClassName("mySlides");
+
+  for (i = 0; i < slides.length; i++) {
```

Comment:
We can move the variable declaration for `i` inside this loop - `for (let i = 0`